alacritty/src/window.rs

```diff
@@ -47,9 +52,14 @@ use crate::gl;
 use crate::wayland_theme::AlacrittyWaylandTheme;
 
 // It's required to be in this directory due to the `windows.rc` file
-#[cfg(not(target_os = "macos"))]
+#[cfg(not(any(target_os = "macos", windows)))]
 static WINDOW_ICON: &[u8] = include_bytes!("../../extra/windows/alacritty.ico");
 
+// This should match the definition of IDI_ICON from `windows.rc`
+#[cfg(windows)]
+const IDI_ICON: WORD = 0x101;
```

Comment:
```suggestion
static IDI_ICON: WORD = 0x101;
```

Reply:
Why the preference for `static` here?

Reply:
I forgot that we really need constexpr like thing here. nvm.